inc/sainexthopgroup.h

```diff
@@ -219,6 +255,30 @@ typedef enum _sai_next_hop_group_member_attr_t
      */
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_MONITORED_OBJECT,
 
+    /**
+     * @brief Object index in the fine grain ECMP table.
+     *
+     * Index specifying the member's order.
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_ONLY
+     * @default internal
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_ATTR_INDEX,
+
+    /**
+     * @brief Object priority for enforcing the members' order.
+     *
+     * Index specifying the member's order.
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_DYNAMIC_ORDERED_ECMP.
```

Comment:
should we add more description how priority value should be given(low to high/ high to low) and how it get mapped in order list

Reply:
Done. High/Low definition does not really matter because within the same SAI implementation the ordering will stay the same, but the mapping description is provided

Reply:
@marian-pritsak  : let say we pass NH1 as 1, NH2 as  10 and NH3 as 7 then SAI will order them as  { NHI, NH3, NH2 }  or {NH2, NH3, NH1} 

basically the number we pass here will form in ascending order or descending order ?  I want to clarify from that perspective.